ex1/ex1-index.js

```diff
@@ -0,0 +1,28 @@
+// **1. Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată**
+// Ar trebui să aibă proprietăți pentru titlu (un șir de caractere), porții (un număr) și ingrediente (o șir de elemente - array). 
+//Va fi afișată pe rânduri separate(o instrucțiune`console.log` pentru fiecare) toate informațiile obiectului după crearea acestuia.
+// Exemplu 1:
+// `Input: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]`
+// `Output: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }`
+//.....................................................................................
+
+// Function to create a recipe object
+function createRecipe(title, portions, ingredients) {
+  // Create an object with the given title, portions, and ingredients
+  let recipe = {
+    title: title, // Property for the recipe title
```

Comment:
(opt) comentariile in general se recomanda sa fie pe o linie noua. De obicei de asupra.